netlify.toml

```diff
@@ -20,20 +20,19 @@
     Referrer-Policy = "origin-when-cross-origin"
     Strict-Transport-Security = "max-age=2592000"
     Permissions-Policy = "vibrate=(), geolocation=(), midi=(), notifications=(), push=(), microphone=(), camera=(), magnetometer=(), gyroscope=(), speaker=()"
-    Basic-Auth = "adapt:letmein2021"
     X-Frame-Options = "DENY"
+    Basic-Auth = "adapt:letmein2021"
 [[headers]]
   for = "/editor"
   [headers.values]
     X-Frame-Options = "ALLOWALL"
-
 # SUBMIT SITEMAP PLUGIN
 # https://www.npmjs.com/package/netlify-plugin-submit-sitemap
 # ###############################################################################
-# [[plugins]]
-# package = "netlify-plugin-submit-sitemap"
-#   [plugins.inputs]
-#     # Path to the sitemap URL (optional, default = /sitemap.xml)
-#     sitemapPath = "/sitemap/sitemap-index.xml"
+[[plugins]]
+package = "netlify-plugin-submit-sitemap"
+  [plugins.inputs]
+    # Path to the sitemap URL (optional, default = /sitemap.xml)
+    sitemapPath = "/sitemap/sitemap-index.xml"
```

Comment:
We should move this into a branch context tied to main.

```suggestion
# Production context:
# All deploys from the main repository branch
# will inherit these settings.
[context.main]
  # Deploys from main branch run this plugin in the build.
  # Plugins context requires double brackets.
  [[context.main.plugins]]
    package = "netlify-plugin-submit-sitemap"
    [context.main.plugins.inputs]
      # Path to the sitemap URL (optional, default = /sitemap.xml)
      sitemapPath = "/sitemap/sitemap-index.xml"
```

Not entirely sure on the syntax for the sitemap plugin inputs but I think that might be it.